### PR TITLE
Update for new op2ext submodule layout

### PR DESCRIPTION
**Pull request by DanRStevens**
Update reference and include settings.

### CODE REVIEW

**Comment by Brett208** [comment]
I couldn't get it to compile from a clean repository clone.

Error	LNK1181	cannot open input file '\NetFixClient2\Debug\op2extDLL.lib'	NetFixClient	\NetFixClient2\LINK	1	

Error	LNK1181	cannot open input file '\NetFixClient2\Debug\op2extStatic.lib'	op2extDLL	\NetFixClient2\op2ext\srcDLL\LINK	1	

It is like it cannot find the new project files for op2ext. Although it appears appveyor found them.

Is this using the current master branch for op2ext? I see the submodule was changed but I'm not sure how to tell what branch the submodule changed to.

Out of time to troubleshoot today.

---

Also, I noticed we are only building the release configuration for NetHelperClient. I'd prefer to build both debug and release unless build time is too long or something as it helps catch configuration bugs.

Thanks,
Brett

**Comment by DanRStevens** [comment]
Edit: This is an update for the recently merged layout change to op2ext. It involves a submodule update.

As the submodule was changed, you may need to re-run `git submodule init` and/or `git submodule update` to bring in the submodule changes. The second command is particularly important. Without explicitly updating the submodule, it will likely try to build with an older version, which will likely result in files or projects not being found.

----

I'll check into the build settings for Debug.

**Comment by Brett208** [comment]
I had to add op2ext's static project to the NetFixClient solution to get this to compile. I don't understand why I have to add this locally, but appveyor does not.

Also, even though it compiles, op2ext is unable to load NetFixClient's dll. It is saying it is at an invalid address. The invalid address is reported by op2ext, not NetFixClient's internal base address checks.

Also, the debugger is no longer picking up op2ext when running debug from NetFixClient.

Out of time for today. Not sure what next step is, but maybe tomorrow it will go better.

-Brett

File: NetFixClient.sln [hunk 1]
```diff
@@ -7,10 +7,12 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "NetFixClient", "NetFixClien
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "OP2Internal", "OP2Internal\src\OP2Internal.vcxproj", "{97594BB1-793F-379B-F6AB-040AC8FE1424}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "op2ext", "op2ext\srcDLL\op2extDLL.vcxproj", "{6DE3610C-C9EE-4FD3-BBD7-66382C16FDE9}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "op2extDLL", "op2ext\srcDLL\op2extDLL.vcxproj", "{6DE3610C-C9EE-4FD3-BBD7-66382C16FDE9}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Outpost2DLL", "op2ext\Outpost2DLL\Outpost2DLL.vcxproj", "{B3F40368-F3CC-4EC5-A758-F5DAD08FEDDC}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "op2extStatic", "..\op2ext\srcStatic\op2extStatic.vcxproj", "{4F82DF84-DE4A-4AE0-9E05-FC5408AE23BA}"
```
Comment (DanRStevens) [inline NetFixClient.sln]: Hmm, "..\". This looks like a familiar problem. :wink:

**Review by DanRStevens — COMMENTED** [review]
Thanks for updating the solution file. Seems I missed a couple things.

----

I've heard the Visual Studio environment compiles things a bit differently than a command line invocation of MSBuild. Apparently different project and reference settings can be used between the two.

----

Not sure about the errors. Did the local modification to OP2Internal `LoadAddr` value get lost during all the upgrades? Might also have been picking up the wrong files due to the path setting being wrong, which might have lead to a bad build.

**Review by Brett208 — APPROVED** [review]
A PR on op2ext when merged will fix all the issues I was having here. Once that is merged in, this should be fine.

Thanks,
-Brett

**Comment by DanRStevens** [comment]
Thanks for investigating and fixing. We'll have to update here with new submodule updates once that fix is in.

**Pull request merged**